### DISCUSSION TITLE
how to use bind for containerlab generated clos fabric?

### DISCUSSION THREAD

**Discussion by nanisyako** [srl-labs/containerlab#1194] [Q&A] [answered]
I want to use 3 tiear clos fabric with ceos but each tier will have its own interface mappings, so how I can mention that in the containerlab generate cli ? basiscally would it be possible to use bind in the command?

```
containerlab gen -n 3tier --kind ceos --nodes 4,2,1

```

**Comment by hellt** [comment, marked as answer]
It is not possible with the generate command, you have to use templated topologies https://containerlab.dev/lab-examples/templated01/
